components/use-cases-card/use-cases-card.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import style from './use-cases-card.module.scss';
+
+type Props = {
+  title: string;
+  text: string | JSX.Element;
+  logo: React.ComponentType;
+  linkUrl?: string;
+  linkText?: string;
```

Comment:
I think for `text` you can just use `string`. In this instance, since the data is coming from your content file, the properties will always be strings here. If you want to render HTML, we'll have to use `dangerouslySetInnerHTML`, which takes a string.

And is it possible to sort these as well as the props destructuring?

Reply:
I had deleted the rederence to the `JSX.Element in the new renamed file.